lib/ghee/api/issues.rb

```diff
@@ -126,6 +126,42 @@ def events(id=nil)
             Ghee::API::Repos::Issues::Events::Proxy.new(connection,prefix)
           end
 
+          def assignees(&block)
+            prefix = "#{path_prefix}/assignees"
+            Ghee::API::Repos::Issues::Assignees::Proxy.new(connection, prefix, &block)
+          end
+        end
+
+        module Assignees
+          class Proxy < ::Ghee::ResourceProxy
+            accept_header "application/vnd.github.cerberus-preview.full+json"
```

Comment:
Not sure if this is how you've handled previews in the past, but hard-coding the preview API in Ghee seems not ideal. I'd rather explicitly require that callers opt into the preview.